lite/doc.go

```diff
@@ -15,9 +15,7 @@ for you, so you can just build nice applications.
 We design for clients who have no strong trust relationship with any Tendermint
 node, just the blockchain and validator set as a whole.
 
-# Data structures
```

Comment:
why remove?

Reply:
I cant have two go-doc headlines right next to each other. I supposed it was better to highlight the title of the following, rather than the general headline. 

We could put a small text between Data structures and SignedHeader. (Unless there is a way to put two headlines together.

Reply:
ok